sale_warehouse_calendar/models/sale_order_line.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from datetime import timedelta
+
+from odoo import models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    def _get_workload(self, customer_lead):
+        # Since the workload is a number of days, it cannot be 0
```

Comment:
I would add a docstring instead of a comment saying that you compute a workload in number of days

Explaining that you convert a delay into a numbers of days (hence the +1) is more understandable than "it cannot be 0" ;-)